aten/src/ATen/native/UpSample.h

```diff
@@ -0,0 +1,249 @@
+#include <math.h>
+
+#include <ATen/ATen.h>
+
+namespace at {
+namespace native {
+
+static inline void check_dim_size(
```

Comment:
A very helpful thing to write here would have been "Corresonds to `THNN_CHECK_DIM_SIZE`"

Reply:
do you mean like a docstring?

Reply:
Nothing so fancy. Just a comment.

Reply:
sounds good. thanks!